specs/posix/pci_devices.table

```diff
@@ -17,13 +17,13 @@ schema([
 ])
 
 extended_schema(LINUX, [
-    Column("pci_class_id", TEXT, "PCI Device class ID in hex format"),
-    Column("pci_subclass_id", TEXT, "PCI Device  subclass in hex format"),
-    Column("pci_subclass", TEXT, "PCI Device subclass"),
     Column("subsystem_vendor_id", TEXT, "Vendor ID of PCI device subsystem"),
     Column("subsystem_vendor", TEXT, "Vendor of PCI device subsystem"),
     Column("subsystem_model_id", TEXT, "Model ID of PCI device subsystem"),
     Column("subsystem_model", TEXT, "Device description of PCI device subsystem"),
+    Column("pci_class_id", TEXT, "PCI Device class ID in hex format"),
+    Column("pci_subclass_id", TEXT, "PCI Device  subclass in hex format"),
+    Column("pci_subclass", TEXT, "PCI Device subclass"),
```

Comment:
Why do we need to change the order here?

Reply:
I really can't remember why; I'll change it.